# -*- coding: utf-8 -*-
# Generated by Django 1.9.1 on 2016-06-15 14:27
from __future__ import unicode_literals

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('dataset', '0006_auto_20160615_1422'),
    ]

    operations = [
        migrations.CreateModel(
            name='DotaLocation',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('file_url', models.TextField(blank=True, help_text='URL of the data at the remote site.', max_length=255, null=True, unique=True, validators=[django.core.validators.URLValidator()])),
                ('file_size', models.IntegerField(blank=True, default=0, help_text='Size of the data in bytes.', null=True)),
                ('file_path', models.TextField(blank=True, db_index=True, help_text='Path of the data at the local site.', null=True)),
                ('thumbnail_url', models.TextField(blank=True, default=None, help_text='URL of the thumbnail at the remote site.', max_length=255, null=True, validators=[django.core.validators.URLValidator()])),
                ('updated', models.DateTimeField(auto_now=True, help_text='Date of last update')),
            ],
            options={
                'db_table': 'dota_location',
            },
        ),
    ]
